### PR TITLE
Change delay to 1 second

### PR DESCRIPTION
**Pull request by Mladen-Robotics**
**### Change delay to 1 second**
I think it is better for the LED to blink with rate of 1 second to increase its life span.

### CODE REVIEW

**Comment by MartinMihalkov** [comment]
I will check it out

File: arduino_blink.ino [hunk 1]
```diff
@@ -7,7 +7,7 @@ pinMode(LED_PIN, OUTPUT);
 
 void loop() {
 digitalWrite(LED_PIN, HIGH);
-delay(500);
+delay(1000);
```
Comment (MartinMihalkov) [inline arduino_blink.ino]: It is good but i prefer longer delay time

**Review by MartinMihalkov — CHANGES_REQUESTED** [review]
I prefer longer delay.For example 3 seconds!

**Comment by MartinMihalkov** [comment]
> I prefer longer delay.For example 3 seconds!

Are you sure

File: arduino_blink.ino [hunk 2]
```diff
@@ -7,7 +7,7 @@ pinMode(LED_PIN, OUTPUT);
 
 void loop() {
 digitalWrite(LED_PIN, HIGH);
-delay(500);
```
Comment (MartinMihalkov) [inline arduino_blink.ino:10]: Change the line here to 3000

**Review by MartinMihalkov — COMMENTED** [review]
Let it be 3 seconds

**Review by MartinMihalkov — COMMENTED** [review]
I am waiting

**Review by MartinMihalkov — CHANGES_REQUESTED** [review]
Make it 3 seconds

**Comment by Mladen-Robotics** [comment]
I changed the blink rate to 3 seconds

**Review by MartinMihalkov — APPROVED** [review]
Now it is perfect!

**Pull request merged**